### PR TITLE
Daml Finance: Upgrade Tutorial

### PR DESCRIPTION
**Pull request by jsjoedin**

### CODE REVIEW

File: docs/2.8.0/docs/daml-finance/tutorials/upgrade/account.rst [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+a `MyAccountUpgradeRule` contract (which is also part of the ``MyAccount.daml`` module):
+
+.. literalinclude:: ../../finance-upgrades/daml/Scripts/UpgradeAccount.daml
+  :language: daml
+  :start-after: -- UPGRADE_ACCOUNT_CREATE_UPGRADE_RULE_BEGIN
+  :end-before: -- UPGRADE_ACCOUNT_CREATE_UPGRADE_RULE_END
+
+Clients Upgrades
+================
+
```
Comment (carrielaben-da) [inline docs/2.8.0/docs/daml-finance/tutorials/upgrade/account.rst:77]:
```suggestion
Clients Upgrade
===============

```

**Review by carrielaben-da — APPROVED** [review]

File: docs/2.8.0/docs/daml-finance/tutorials/upgrade/intro.rst [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+#################
+
+This section delves into the processes for upgrading existing contract instances on a Daml ledger.
+A thorough understanding of the different types of upgrades is essential to choose the most suitable
+method for various scenarios.
+
+Upgrades are broadly classified into two categories:
+
+* *Voluntary Upgrades:*
+  These upgrades only happen if you act, akin to an airline offering an upgrade from economy to
```
Comment (markus-da) [inline docs/2.8.0/docs/daml-finance/tutorials/upgrade/intro.rst:14]:
```suggestion
  These upgrades only happen if the holding owner acts, akin to an airline offering an upgrade from economy to
```

File: docs/2.8.0/docs/daml-finance/tutorials/upgrade/intro.rst [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  business class. The upgrade materializes only if you opt-in and make the necessary payment.
+
+* *Mandatory Upgrades:*
+  These upgrades happen inevitably, regardless of your preference. A common example is the automatic
```
Comment (markus-da) [inline docs/2.8.0/docs/daml-finance/tutorials/upgrade/intro.rst:18]:
```suggestion
  These upgrades happen inevitably, regardless of the preference of the holding owner. A common example is the automatic
```

File: docs/2.8.0/docs/daml-finance/tutorials/upgrade/intro.rst (hunk 2 above)
Comment (markus-da) [reply docs/2.8.0/docs/daml-finance/tutorials/upgrade/intro.rst:14]: When I first read this I was a bit unsure, until I realized what _you_ was referring to.

**Review by markus-da — APPROVED** [review]
LGTM, left a couple of minor suggestions.

File: docs/2.8.0/docs/daml-finance/tutorials/upgrade/intro.rst (hunk 2 above)
Comment (johan-da) [reply docs/2.8.0/docs/daml-finance/tutorials/upgrade/intro.rst:14]: I modified it to make it more clear.

**Pull request merged**